meta/documents/user_guide/en/initial_setup.md

```diff
@@ -21,41 +21,57 @@
 After the plugin is installed in your Plentymarkets Plugin Set, the plugin must be configured to use the correct credentials when connecting to Wayfair's interfaces.
 
 * **The authorization procedure must be performed for any Plugin Set that contains the Wayfair plugin**.
+
 * Copying a Plugin Set will copy the authorization information to the new plugin set.
+
 * An exported or imported Plugin Set may include the authorization information.
 
 The authorization steps are as follows:
-1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`:
+
+    ![plugins_menu_plugin_set_overview](../../../images/en/plugins_menu_plugin_set_overview.png)
+
+2. Locate the Plugin Set that is linked to the client with which Wayfair will be used:
 
-2. Locate the Plugin Set that is linked to the client with which Wayfair will be used.
+    ![linked clients](../../../images/en/plugin_sets_linked_clients.png)
 
-3. Click on the `Edit` button for the desired Plugin set
+3. Click on the `Edit` button ![pencil button](../../../images/common/button_pencil.png) for the desired Plugin set.
 
-4. In the Wayfair row of the Plugin set, click on the `Settings` button.
+4. In the Wayfair row of the Plugin set, click on the `Settings` button ![gear button](../../../images/common/button_gear.png).
 
-4. In the left-side menu, go to `Configuration` >> `Global Settings`.
+4. In the left-side menu, go to `Configuration` >> `Global Settings`:
 
-5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials
+    ![global settings in menu](../../../images/en/menu_global_settings.png)
 
-6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md)
+5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials.
 
-7. Click the `Save` button in the toolbar above the settings
+6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md):
+
+    ![global settings live](../../../images/en/global_settings_live.png)

```

Comment:
Highlight the the "Mode *" dropdown so it is 100% clear where they have to change the Mode to `Live`